tests/integration_test/jwt_locations_test/jwt_locations_test.go

```diff
@@ -0,0 +1,179 @@
+// Copyright 2019 Google LLC
```

Comment:
just merge this tests into jwt_auth_integration_test.go.

We have too many tests folders now, which is totally unnecessary.

Reply:
Gotcha. 

BTW, our tests folders are totally unnecessary since we can still make our integration tests without put into different library. I suggest we can move them out of the test folders into one simple integration_test folder like before. Are you OK with that?

Reply:
> We have too many tests folders now, which is totally unnecessary.

This is so the tests run in parallel. Each folder runs all the tests as one process. Currently all tests run in 2 minutes on my desktop, which is very useful when developing.

> I suggest we can move them out of the test folders into one simple integration_test folder like before

If this is just to make our folder structure look nice, I don't agree. I would rather not wait 25 minutes for tests to run.

Reply:
> > We have too many tests folders now, which is totally unnecessary.
> 
> This is so the tests run in parallel. Each folder runs all the tests as one process. Currently all tests run in 2 minutes on my desktop, which is very useful when developing.
> 
> > I suggest we can move them out of the test folders into one simple integration_test folder like before
> 
> If this is just to make our folder structure look nice, I don't agree. I would rather not wait 25 minutes for tests to run.

still make our integration tests *parallel*

As what I see, there is no difference between putting tests into different package and manually setting t.parallel with `go test -p `

https://medium.com/@xcoulon/how-to-avoid-parallel-execution-of-tests-in-golang-763d32d88eec
https://stackoverflow.com/questions/24375966/does-go-test-run-unit-tests-concurrently

Reply:
I see, could you try it out on our repo and make sure it works? If they still run in parallel, then that's definitely cleaner.

Reply:
> just merge this tests into jwt_auth_integration_test.go.
> 
> We have too many tests folders now, which is totally unnecessary.

The jwt_authn already has two tests and it maybe better to jwt_location in a separate file. 

Will remove folders in separate PR.